LabController/src/bkr/labcontroller/expire_distros.py

```diff
@@ -78,10 +78,12 @@ def check_url(url):
 def check_all_trees(ignore_errors=False,
                     dry_run=False,
                     lab_controller='http://localhost:8000',
-                    remove_all=False):
+                    remove_all=False,
+                    filter=None):
+    filter_on_arch = True if (filter is not None and filter and 'arch' in filter.keys()) else False
```

Comment:
```suggestion
    filter_on_arch = filter and 'arch' in filter
```

Reply:
Perhaps we could streamline this further by converting `filter` to `dict` if it is set to `None`.